README.md

```diff
@@ -10,9 +10,51 @@ Istio is an open-source service mesh that provides a uniform way to manage, conn
 
 The Istio module allows you to add Istio Operator to the Kyma runtime. Within Istio Operator, Istio Controller is responsible for installing, uninstalling, and managing Istio. For more information, read the [Istio Controller documentation](./docs/user/00-10-overview-istio-controller.md).
 
+## Install Istio Manager and Istio from the latest release
+
+### Prerequisites
+
+- Access to a Kubernetes (v1.24 or higher) cluster
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+### Procedure
+
+1. To install Istio, you must install latest Istio Manager first. Run the following:
```

Comment:
The file contains both Deployment, CRD and RBAC for the controller. Maybe the name of the file could also be adjusted?
```suggestion
1. To install Istio, you must install latest Istio Manager and Istio CRD first. Run the following:
```

Reply:
hmm, they are all needed for the `istio-manager` as whole component entity. do you have better name for the file?

Reply:
If you are fine with the name it's OK, I would only update the documentation then that it contains all the needed stuff

Reply:
Personally I prefer `istio-operator.yaml`, but to align to KEDA and other we should stick to manager I guess.